src/clr-angular/forms/combobox/_combobox.clarity.scss

```diff
@@ -1,54 +1,128 @@
-// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
 @import '../../popover/dropdown/menu-mixins.clarity';
 @import './properties.combobox';
 
 @include exports('combobox.clarity') {
-  .clr-combobox {
-    display: inline-block;
+  .clr-aria-only {
+    display: none;
   }
 
-  .clr-combobox-input-container {
-    display: flex;
-    align-items: center;
-  }
+  .clr-combobox-container {
+    position: relative;
 
-  .clr-combobox-input {
     //Dimensions
     padding: 0 $clr-ng-select-trigger-right-padding 0 $clr-ng-select-trigger-left-padding;
     min-height: $clr-ng-select-min-height;
-    width: $clr-ng-select-min-width;
+    min-width: $clr-ng-select-min-width;
     @include css-var(font-size, clr-ng-select-font-size, $clr-ng-select-font-size, $clr-use-custom-properties);
 
+    &.multi {
+      min-width: $clr-ng-multiselect-min-width;
+      padding-bottom: $clr_baselineRem_0_125;
+    }
+
     //Mixins
-    @include form-fields-appearance();
-    @include input-border-bottom-animation();
+    @include form-fields-appearance($clr-forms-border-color);
+
+    // override the above mixin for combobox-specific styles
+    background: hsl(201, 10%, 96%); // #F4F5F6
```

Comment:
For theming purposes I think this should be a variable.

Reply:
Thanks for this, Matt! It brought to my attention the fact that we were far behind with the Dark Theme. Thanks to you and Colin's quickly provided spec, we now have a working one.
Fixed!